account_multicompany_easy_creation/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2018 Carlos Dauden - Tecnativa <carlos.dauden@tecnativa.com>
+# Copyright 2022 Moduon - Eduardo de Miguel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Quick Company Creation Wizard",
+    "summary": "This module adds a wizard to create companies easily",
+    "version": "16.0.1.0.0",
+    "category": "Multicompany",
+    "website": "https://github.com/OCA/multi-company"
+    "account_multicompany_easy_creation",
```

Comment:
This seems a bug. Not yours, but can you fix it now that you're at it please?

```suggestion
    "website": "https://github.com/OCA/multi-company",
```